services-directions/src/test/java/com/mapbox/api/directions/v5/models/RouteOptionsTest.java

```diff
@@ -192,6 +193,119 @@ public void directionsRequestResult_doesContainOverview() throws Exception {
     assertEquals(DirectionsCriteria.OVERVIEW_SIMPLIFIED, route.routeOptions().overview());
   }
 
+  @Test
+  public void directionsRequestResult_doesContainBicycleType() throws Exception {
+    Response<DirectionsResponse> response = MapboxDirections.builder()
+            .geometries(DirectionsCriteria.GEOMETRY_POLYLINE6)
+            .baseUrl(mockUrl.toString())
+            .accessToken(ACCESS_TOKEN)
+            .origin(Point.fromLngLat(1.0, 1.0))
+            .destination(Point.fromLngLat(5.0, 5.0))
+            .profile(DirectionsCriteria.PROFILE_WALKING)
+            .continueStraight(false)
+            .language(Locale.CANADA)
+            .bicycleType(ROAD)
+            .alternatives(true).build().executeCall();
+    DirectionsRoute route = response.body().routes().get(0);
+
+    assertEquals(ROAD, route.routeOptions().bicycleType());
```

Comment:
Minor thing - The "expected" values are better represented using the literal value instead of using the constant (👀 [Expect Literals](https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=61)). The same thing applies to `directionsRequestResult_doesContainCyclingSpeed`, `directionsRequestResult_doesContainUseRoads`, `directionsRequestResult_doesContainUseHills`, `directionsRequestResult_doesContainUseFerry` and `directionsRequestResult_doesContainAvoidBadSurfaces` tests.